webapps/ui/admin/client/scripts/config.js

```diff
@@ -49,6 +49,9 @@ export default {
   //     'unit': 'day'
   //   }
   // },
+  // 'uniqueTaskWorkersMetricToggle': {
+  //   'display': false
+  // },
```

Comment:
Do we plan to add more options for the toggle? If not, let's make this a boolean. We could also think about a more expressive name for the config option:

```suggestion
  // 'showTaskWorkerMetrics': true,
```

Reply:
We don't plan to add more options for this. I renamed it to `alwaysShowUniqueTaskWorkerMetrics`, defaults to true.
If it's true we don't display the toggle and always fetch UTW.
If it's false we display the toggle and by default don't fetch UTW.